data/moves.ts

```diff
@@ -15121,6 +15121,18 @@ export const Moves: {[moveid: string]: MoveData} = {
 		pp: 10,
 		priority: 0,
 		flags: {protect: 1, mirror: 1},
+		onTryMove() {
+			this.attrLastMove('[still]');
```

Comment:
I'm a bit uncomfortable with the animation removal and the animation adding to be on two separate events. Why aren't these both in `onPrepareHit`?

Also `attrLastMove('[anim] Poison Jab')` works, right?

Reply:
Also, may want to `attrLastMove('[anim] Shell Side Arm Physical')` and then alias it to Poison Jab in `move-animations`.

Reply:
> attrLastMove('[anim] Shell Side Arm Physical')

wait thats a thing????

Reply:
I just copied the staff bros behavior tbh. And yes `attrLastMove('[anim] Poison Jab')` does work.

Reply:
@xInstruct lol yes Staff Bros should probably be using it, too.